SparkPost/2web2ui
# SA-2310: Delayed rate metric is not calculated correctly

### What Changed

- Changed "Delayed Rate" metric to use "Delayed 1st Attempt" metric

### How To Test

- Open Report Builder
- Select "Accepted", "Delayed", "Delayed 1st Attempt", and "Delayed Rate" metrics
- Confirm "Delayed Rate" = "Delayed 1st Attempt" / "Accepted"
